cmake/scripts/android/ArchSetup.cmake

```diff
@@ -17,6 +17,7 @@ else()
   if(CPU STREQUAL armeabi-v7a)
     set(ARCH arm)
     set(NEON True)
+    set(ENABLE_ISO9660PP OFF)
```

Comment:
maybe needs this syntax?
https://github.com/xbmc/xbmc/blob/b8fe3f314de594c09dc7c94cd5bed7ffc0fe59c2/cmake/scripts/android/ArchSetup.cmake#L38

Reply:
Perfect, thanks